pkgs/development/interpreters/clips/default.nix

```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl }:
 
 stdenv.mkDerivation {
-  version = "6.30";
+  version = "6.31";
   pname = "clips";
   src = fetchurl {
-    url = "mirror://sourceforge/clipsrules/CLIPS/6.30/clips_core_source_630.tar.Z";
-    sha256 = "1r0m59l3mk9cwzq3nmyr5qxrlkzp3njls4hfv8ml85dmqh7n3ysy";
+    url = "mirror://sourceforge/clipsrules/CLIPS/6.31/clips_core_source_630.tar.Z";
```

Comment:
```suggestion
    url = "mirror://sourceforge/clipsrules/CLIPS/6.31/clips_core_source_631.tar.Z";
```

Reply:
We should try to derive that fragment from the version to support auto update.